sites/svelte.dev/package.json

```diff
@@ -48,7 +48,7 @@
 		"shelljs": "^0.8.5",
 		"shiki": "^0.14.2",
 		"shiki-twoslash": "^3.1.2",
-		"svelte": "^3.59.1",
+		"svelte": "workspace:*",
```

Comment:
it's an interesting question whether we'd want to make this `workspace` or not. I wonder for now if maybe it's safer just to stick to a specific released version?

Reply:
workspace:* helps because the documentation will always be synced with the current Svelte APIs. Not to mention, I'm preparing the site for v4, and unable to do that without going `workspace:*` 😅

Reply:
Ah, ok